packages/wonder-blocks-tokens/src/theme/semantic/semantic-color.ts

```diff
@@ -245,23 +305,23 @@ export const semanticColor = {
      */
     status: {
         critical: {
-            background: color.fadedRed8,
-            foreground: color.red,
+            background: core.destructive.subtle,
```

Comment:
Should we have `critical` and `notice` core tokens so that it is separate from `destructive` and `progressive` tokens?